etc/WMAgentConfig.py

```diff
@@ -47,11 +47,10 @@
 agentName = "WMAgentCommissioning"
 agentNumber = 0
 
-# List of BossAir plugins that this agent will use.
-bossAirPlugins = ["CondorPlugin"]
-
-## Plugin with Condor-Python API ...
+# Plugin with Condor-Python API ...
 bossAirPlugins = ["PyCondorPlugin", "CondorPlugin"]
```

Comment:
As far as I remember there was a bug if we specify both try to do something in PyCondorPlugin. So probably we can change this a little later?